agent/pkg/dagent/utils/docker.go

```diff
@@ -283,6 +285,8 @@ func logDeployInfo(dog *dogger.DeploymentLogger, deployImageRequest *v1.DeployIm
 	}
 }
 
+//nolint
+// TODO(polaroi8d): DeployImage function is too long for golangci-lint (105 > 100)
```

Comment:
Not cool.  I would move all the label manipulation related things into a separate function, to save some length.